src/Microsoft.Health.Dicom.Functions.Client/Configs/OperationRoutesConfiguration.cs

```diff
@@ -3,12 +3,16 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
+using System;
 using System.ComponentModel.DataAnnotations;
 
 namespace Microsoft.Health.Dicom.Functions.Client.Configs
 {
     public class OperationRoutesConfiguration
     {
+        [Required]
+        public Uri StartQueryTagReindex { get; set; }
```

Comment:
The problem is that the template is a format string, while some of the other fields are simply routes. Because they're in a route template, I felt it was redundant to call them routes, but I could use the terms. Eg.
```csharp
public Uri FooRoute { get; set; }

public string BarRouteTemplate { get; set; }
```

Reply:
I like the new way. :)

Reply:
I'll got that "route" 😛